src/VirtoCommerce.Platform.Web/Licensing/LicenseProvider.cs

```diff
@@ -30,5 +73,27 @@ public License GetLicense()
 
             return license;
         }
+
+        public void SaveLicense(License license)
+        {
+            using (var stream = _blobStorageProvider.OpenWrite(GetLicenseFileUrl()))
+            {
+                var streamWriter = new StreamWriter(stream);
+                streamWriter.Write(license.RawLicense);
+                streamWriter.Flush();
+            }
+        }
+
+
+        protected string GetLicenseFileUrl()
```

Comment:
I think these one-line  methods also can be replaced to using in-place (just note)